### ISSUE TITLE
请问一下训练deeplabv3的时候学习率为啥会变为复数 assert isinstance(v, (float, int))

### ISSUE THREAD

**Issue by fitzBaruch** [WZMIAOMIAO/deep-learning-for-image-processing#625]
**System information**
* Have I written custom code:
* OS Platform：Linux Ubuntu 20.04
* Python version:
* Deep learning framework and version ： Pytorch1.12
* Use GPU or not: GPU
*CUDA/cuDNN version(if you use GPU):CUDA 11.7 or CUDA 11.3
* The network you trained： deeplabv3

训练着训练着，学习率会从float变成complex(复数) ，log信息如下

**Error info / logs**
File "/home/ubuntu/project/deeplab_v3/train_multi_GPU.py", line 167, in main
    mean_loss, lr = train_one_epoch(model, optimizer, train_data_loader, device, epoch,
  File "/home/ubuntu/project/deeplab_v3/train_utils/train_and_eval.py", line 60, in train_one_epoch
    metric_logger.update(loss=loss.item(), lr=lr)
  File "/home/ubuntu/project/deeplab_v3/train_utils/distributed_utils.py", line 135, in update
    assert isinstance(v, (float, int))
AssertionError

**Comment by WZMIAOMIAO** [comment]
不好意思，没遇到过你说的问题哦。
